artemis-distribution/src/main/assembly/dep.xml

```diff
@@ -106,6 +110,27 @@
             <include>com.sun.xml.bind:jaxb-core</include>
             <include>jakarta.activation:jakarta.activation-api</include>
             <include>jakarta.security.auth.message:jakarta.security.auth.message-api</include>
+            <!-- quorum -->
```

Comment:
Where are the zk deps here too?

Reply:
https://github.com/apache/activemq-artemis/pull/3555/files#r625528887